### PR TITLE
more typos and polishing for Service systems

### PR DESCRIPTION
**Pull request by adaortizcarbonell**

### CODE REVIEW

**Comment by adaortizcarbonell** [comment]
As for all the "typosX" PRs, please consider them just as suggestions. I don't want to change the original meaning.

File: content/services/systems/_index.md [hunk 1]
```diff
@@ -8,37 +8,26 @@ tag: systems
...
-shiny things. Developing software systems in a maintainable way with focus on
-correctness then becomme a nice challenge. And we are up to the it - our
-software engineers usually consider the case more interesting, the more complex it gets.
+In today's world of everything distributed, making a system do
+the right thing is not always an easy task. In addition, requirements constantly evolve and business owners often like to use the newest and shiniest advancements. Developing software systems in a maintainable manner with focus on correctness becomes then a challenge. And we are up for that challenge - our software engineers usually consider the case more interesting, the more complex it gets.
 
 <!--more-->
 
-Consultants at Expert Analytics have participated in large systems devlopment projects
+Consultants at Expert Analytics have participated in large system development projects
```
Comment (mandus) [inline content/services/systems/_index.md]: I think it should actually be `systems development` here

File: content/services/systems/_index.md [hunk 2]
```diff
@@ -8,37 +8,26 @@ tag: systems
 anchor: systems
 weight: 100
 ---
-In todays world of everything distributed, making a system do
-the right thing is not always an easy task. Adding to that, requirements that
-constantly evolve, and business owners who often like to use the new and
-shiny things. Developing software systems in a maintainable way with focus on
-correctness then becomme a nice challenge. And we are up to the it - our
-software engineers usually consider the case more interesting, the more complex it gets.
+In today's world of everything distributed, making a system do
```
Comment (mandus) [inline content/services/systems/_index.md]: I like this rewording!

File: content/services/systems/_index.md [hunk 3]
```diff
@@ -8,37 +8,26 @@ tag: systems
...
 we understand that a tool is very wrong for the problem to be solved, and then
 we will speak out.
 
-To get the full picture of the languages, tools, and environments that Expert
-Analytics can use for your project, the individual CVs of our employees must be
-consulted. What we most often use is Python, Java, Go, and C++. We have worked
-extensively with Postgres, Azure SQL, Google Datastore and Firestore, and Cosmos
-DB. For continous testing and integration we are familiar with most of the main
+To get the full scope of the languages, tools, and environments that Expert
+Analytics can apply to your project, the individual CVs of our employees can be consulted. What we most often use is Python, Java, Go, and C++. We have worked extensively with Postgres, Azure SQL, Google Datastore and Firestore, and Cosmos DB. For continuous testing and integration we are familiar with most of the main
```
Comment (mandus) [inline content/services/systems/_index.md]: If I may, I like to keep the phrase `must be consulted` - or feel this conveys a slightly different meaning (that it is impossible to show everything here and the list that follows may not be the most important for a possible reader..) Besides that I like the other changes

File: content/services/systems/_index.md (hunk 1 above)
Comment (adaortizcarbonell) [reply content/services/systems/_index.md]: done

File: content/services/systems/_index.md (hunk 3 above)
Comment (adaortizcarbonell) [reply content/services/systems/_index.md]: Aha, now I understand the meaning you intended. At first "must" sounded a bit strong, but now I get it.

**Pull request merged**